frontend/elements/src/entry/home/home/search-results/search-result.ts

```diff
@@ -15,6 +15,30 @@ export class _ extends LitElement {
     static get styles() {
         return [
             css`
+                @keyframes jump {
+                    0% {
+                        transform: scaleX(-1) translateY(0px);
+                    }
+                    20% {
+                        transform: scaleX(-1) translateY(0px);
+                    }
```

Comment:
We need to wait a bit for the card flip animation to complete before animating the jiggling into view